### DISCUSSION TITLE
Hello World problem. (a target file must be specified)

### DISCUSSION THREAD

**Discussion by BladePoint** [BowlerHatLLC/vscode-as3mxml#620] [Q&A] [answered]
I am following [these instructions](https://github.com/BowlerHatLLC/vscode-as3mxml/wiki/Create-a-new-ActionScript-project-in-Visual-Studio-Code-that-targets-Adobe-Flash-Player) to create a SWF file that displays "Hello world". I can produce a Main.swf file that does exactly that when I compile it, but in the VS Code Problems panel, I'm still getting this message:

```
∨ {} asconfig.json HelloWorld
  a target file must be specified. (1319) [Ln 1, Col 1] ^
    C:\Users\bladepoint\VSCode\HelloWorld\asconfig.json (line: -1)
```

What does this error message mean and how can I fix it?

**Comment by joshtynjala** [comment]
Please share the contents of your _asconfig.json_ file.

**Comment by BladePoint** [comment, marked as answer]
I started over and followed the guide from the beginning. The error message has disappeared.
